maxim/logger/fireworks/utils.py

```
@@ -0,0 +1,347 @@
+"""Utility functions for Fireworks AI API integration with Maxim.
+
+This module provides utility functions for parsing and processing Fireworks AI API
+requests and responses to integrate with Maxim's logging and monitoring system.
+It handles message format conversion, parameter extraction, response standardization,
+and image attachment processing for multimodal support.
+"""
+
+import time
+from typing import Any, Dict, Iterable, List 
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)